apps/web/src/ee/clerk/providers/ClerkProvider.tsx

```diff
@@ -5,54 +5,103 @@ import { ClerkProp, ClerkProvider as _ClerkProvider } from '@clerk/clerk-react';
 import { useColorScheme } from '@novu/design-system';
 import { dark } from '@clerk/themes';
 import { buildClerk } from './clerk-singleton';
+import { Variables } from '@clerk/types';
 
-const ClerkModalElement = {
-  modalContent: {
-    width: '80rem',
-    display: 'block',
-  },
-  modalBackdrop: {
-    zIndex: 200,
-  },
-  cardBox: {
-    width: '100%',
-  },
-  rootBox: {
-    width: 'auto',
-  },
-  navbarButton: {
-    fontFamily: 'var(--nv-fonts-system)',
-    fontWeight: 'var(--nv-font-weights-strong)',
-    fontSize: 'var(--nv-font-sizes-88)',
-    paddingTop: 'var(--nv-spacing-50) !important',
-    paddingBottom: 'var(--nv-spacing-50) !important',
-    color: 'var(--nv-colors-typography-text-secondary) !important',
-    '&:hover, &:focus, &:active': {
-      color: 'var(--nv-colors-typography-text-main) !important',
-    },
-  },
-  navbarButtonIcon: {
-    height: 'var(--nv-sizes-125)',
-    width: 'var(--nv-sizes-125)',
-  },
-};
-
-const localization = {
+const CLERK_LOCALIZATION = {
   userProfile: {
     navbar: {
       title: 'Settings',
       description: '',
+      account: 'User profile',
+      security: 'Access security',
     },
   },
+
+  userButton: {
+    action__signOut: 'Log Out',
+  },
 };
 
 const ALLOWED_REDIRECT_ORIGINS = ['http://localhost:*', location.origin];
 
 export const ClerkProvider: React.FC<PropsWithChildren<{}>> = ({ children }) => {
+  const { colorScheme } = useColorScheme();
+
+  const CLERK_MODAL_ELEMENT = {
+    modalContent: {
+      width: '80rem',
+      display: 'block',
+    },
+    modalBackdrop: {
+      zIndex: 200,
```

Comment:
<!-- Chores are simple tasks that must be done before the subject can be “officially” accepted. Usually, these comments reference some common process. Try to leave a link to the process description so that the reader knows how to resolve the chore.  -->
☑ **chore:** Please use `zIndex` token‏

Reply:
opted to skip for now